File: dongyimai-manager-web/src/main/webapp/js/service/TbContentCategoryService.js
//内容分类服务层
app.service("TbContentCategoryService",function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.post("/TbContentCategoryController/findAll");
	}
	//分页 
	this.findPage=function(pageNum,pageSize,tbContentCategory){
		return $http.post("/TbContentCategoryController/findPage?pageNum="+pageNum+"&pageSize="+pageSize,tbContentCategory);
	}
	//查询实体
	this.findOne=function(id){
		return $http.post("/TbContentCategoryController/findOne?id="+id);
	}
	//增加、修改
	this.save=function (tbContentCategory) {
		let url;
		if(tbContentCategory.id==null){
			url="/TbContentCategoryController/add";
		}else {
			url="/TbContentCategoryController/update"
		}
		return $http.post(url,tbContentCategory);
	}
	//删除
	this.delete=function(ids){
		return $http.post("/TbContentCategoryController/delete?ids="+ids);
	}
});

File: dongyimai-interface/src/main/java/org/example/liuhengfei/service/TbContentCategoryService.java
package org.example.liuhengfei.service;

import org.example.liuhengfei.dto.PageResult;
import org.example.liuhengfei.pojo.TbContentCategory;

import java.util.List;
/**
 * 内容分类服务层接口
 * @author Administrator
 *
 */
public interface TbContentCategoryService {

	/**
	 * 返回全部列表
	 * @return
	 */
	List<TbContentCategory> findAll();
	
	/**
	 * 返回分页列表
	 * @return
	 */
	PageResult findPage(int pageNum,int pageSize,TbContentCategory tbContentCategory);

/**
 * 根据ID获取实体
 * @param id
 * @return
 */
TbContentCategory findOne(Long id);

	/**
	 * 增加
	*/
	int add(TbContentCategory tbContentCategory);

	/**
	 * 修改
	 */
	int update(TbContentCategory tbContentCategory);
	
	/**
	 * 批量删除
	 * @param ids
	 */
	int delete(Long[] ids);
	
}
